interface/ElectronNtuple.h

```diff
@@ -238,6 +240,10 @@ class ElectronNtuple {
 	float ele_mvaIdV1_ = -2.;
 	float ele_mvaIdV2_ = -2.;
 	float ele_conv_vtx_fit_prob_ = -1.;
+	float ele_iso01_ = 0.;
+	float ele_iso02_ = 0.;
+	float ele_iso03_ = 0.;
+	float ele_iso04_ = 0.;
```

Comment:
@mverzett, default these to a nonphysical -ve number? (I don't have a strong opinion)